arangod/Aql/RestAqlHandler.cpp

```diff
@@ -873,8 +873,24 @@ RestStatus RestAqlHandler::handleFinishQuery(std::string const& idString) {
 
 RequestLane RestAqlHandler::lane() const {
   if (ServerState::instance()->isCoordinator()) {
+    // continuation requests on coordinators will get medium priority,
+    // so that they don't block query parts elsewhere
+    TRI_ASSERT(PriorityRequestLane(RequestLane::CLUSTER_AQL_INTERNAL_COORDINATOR) == RequestPriority::MED);
```

Comment:
If you make `PriorityRequestLane` `constexpr`, this assertion can be changed into a `static_assert`. Same for the other asserts below.

Reply:
Done